products/bash/release-products.sh

```diff
@@ -349,6 +349,58 @@ function release_event_streams {
     helm install ${chart_repo}/${event_streams_chart} --name ${event_streams_release_name} --namespace ${event_streams_namespace} --values event-streams-values.yaml --tls
 }
 
+# --- Tracing ----------------------------------------------------------
+
+tracing_release_name=tracing-demo
+tracing_namespace=tracing
+tracing_chart=ibm-icp4i-tracing-prod
+tracing_pull_secret=ibm-entitlement-key
+tracing_storage=ibmc-block-gold
+navigator_namespace=integration
+stringToReplace=icp-console
+proxyString=icp-proxy
+navigatorString=navigator-integration
+proxyHost=${cp_console/$stringToReplace/$proxyString}
+navigatorHost=${cp_console/$stringToReplace/$navigatorString}
```

Comment:
I'm just being picky here because it's generally well done but personally I find the variables in the substitution make it slightly harder to follow, and it would be clearer to write:
```sh
proxyHost=${cp_console/icp-console/icp-proxy}
navigatorHost=${cp_console/icp-console/navigator-integration}
```
But it's a matter of taste.

Reply:
Done